Test/Mono.Cecil.Tests/ImageReadTests.cs

```diff
@@ -207,6 +207,14 @@ public void Net471TargetingAssembly ()
 			});
 		}
 
+		[Test]
+		public void LocallyScopedConstantArray ()
+		{
+			TestModule ("LocallyScopedConstantArray.dll", module => {
+				Assert.IsTrue (module.HasDebugHeader);
```

Comment:
We'd need to assert that the constant is indeed null in there.

Reply:
done